continuous-integration/devcontainers/terraform/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,67 @@
+FROM golang:1.13-stretch
+
+# Avoid warnings by switching to noninteractive
+ENV DEBIAN_FRONTEND=noninteractive
+
+ARG TERRAFORM_VERSION=0.12.24
```

Comment:
One thing I like to do with dev containers is add a label for each tool/version.  Just a little extra metadata for discoverability.  Something like:

LABEL TERRAFORM_VERSION=$TERRAFORM_VERSION
LABEL TFLINT_VERSION =$TFLINT_VERSION